docs/en/transform/sql.md

```diff
@@ -56,7 +56,6 @@ Use SQL plugin for data aggregation, use avg functions to perform aggregation op
 ```bash
 sql {
     sql = "select avg(age) from user_info",
-    table_name = "user_info"
```

Comment:
change `table_name` to `result_table_name` ?

Reply:
> change `table_name` to `result_table_name` ?

thank you,in common-options.mdx, there is a pair of results_ table_ name,source_ table_name unified description of name,I think this parameter should be added to all Sql Transforms or not. What do you think
http://seatunnel.incubator.apache.org/docs/2.3.0-beta/transform/common-options

Reply:
OK, remove it is fine.